xbmc/cores/VideoPlayer/DVDDemuxers/DVDDemux.h

```diff
@@ -243,29 +243,29 @@ class CDVDDemux
   /*
    * Reset the entire demuxer (same result as closing and opening it)
    */
-  virtual void Reset() = 0;
+  virtual void Reset() { }
 
   /*
    * Aborts any internal reading that might be stalling main thread
    * NOTICE - this can be called from another thread
    */
-  virtual void Abort() = 0;
+  virtual void Abort() { }
 
   /*
    * Flush the demuxer, if any data is kept in buffers, this should be freed now
    */
-  virtual void Flush() = 0;
+  virtual void Flush() { }
 
   /*
    * Read a packet, returns NULL on error
    *
    */
-  virtual DemuxPacket* Read() = 0;
```

Comment:
because function is not universally required: https://github.com/xbmc/xbmc/pull/12250/files#diff-3b3e940dfa4c42a41d871daaaabe3d70L37